src/classes/STG_InstallScript.cls

```diff
@@ -338,24 +345,37 @@ global without sharing class STG_InstallScript implements InstallHandler {
         }
     }
     
-    /**********************************************************************************************************
-    * @description Enables prevent deletion by default on Hierarchy Custom Settings for new installs and 
-    * when upgrading to 1.95
+    /************************************************************************************************************************************************************************
+    * @description Enables Prevent Account Deletion and Plan Requirement Deletion by default on Hierarchy Custom Settings for new installs and when upgrading to 1.95
     * @return void
     */
     global static void enablePreventDeletionHierarchyCustomSettings(Version previousVersion) {
-        if ((previousVersion == NULL) || (previousVersion != null && previousVersion.compareTo(new Version(1,95)) < 0)) {
+        if ((previousVersion == null) || (previousVersion != null && previousVersion.compareTo(new Version(1,95)) < 0)) {
```

Comment:
I don't feel that we need the previousVersion != null check in the logic as in the logic flow previousVersion equals null is evaluated first, which would evaluate to true and proceed to the code in the block. Any condition after that is implicitly previousVersion not equals null. I realize this is a style fix, but thought it worth the comment. Not a required change.

Reply:
After a further review on this, I feel like we might be well-served by having a private static helper method that performs this kind of check consistently and makes the code more readable, taking the version as an argument.